R/pkg/R/functions.R

```diff
@@ -3298,6 +3298,42 @@ setMethod("lead",
             column(jc)
           })
 
+  #' @details
```

Comment:
@zero323 . It's weird. Which editor are you using now? This is still wrong. You should not put a leading space here. This line still have 2-spaces at the beginning

Reply:
Remote Atom with somewhat hacky Teletype setup, and it is not like it's R support is great in the first place.

But I should double check this, instead of trusting the thing. Sorry for the mess.

Reply:
Got it. Never mind~ I was just wondering the type of editor, @zero323 . :)